crates/sui-protocol-config/src/lib.rs

```diff
@@ -1671,6 +1672,9 @@ impl ProtocolConfig {
                     // enable nw cert v2 on mainnet
                     cfg.feature_flags.narwhal_certificate_v2 = true;
                 }
+                33 => {
```

Comment:
Do we need 33? I believe this should work in 32 -- 32 hasn't hit testnet yet (and from the release calendar I believe it shouldn't until next week).

Reply:
But are we going to do another branch cut this week? If not, I guess I need to cherry-pick? @ebmifa

Reply:
`testnet` is on protocol version `31`
```
const MAX_PROTOCOL_VERSION: u64 = 31;
```

Reply:
main/devnet is on `32`

Reply:
Are will going to do another branch cut this week for next week's release?

Reply:
Yes, we have a v1.15.0 branch cut this Friday. (Last release of the year)